src/com/untamedears/JukeAlert/util/OnlineGroupMembers.java

```diff
@@ -148,7 +149,7 @@ private Player getNextModerator() {
             GroupPermission perm = manager_.getPermissionforGroup(g);
             List<OfflinePlayer> mods = new ArrayList<OfflinePlayer>();
             for (UUID uuid: uuids)
-            	if (perm.isAccessible(g.getPlayerType(uuid), PermissionType.BLOCKS) && !g.isOwner(uuid))
+            	if (g.getPlayerType(uuid) != PlayerType.MEMBERS && !g.isOwner(uuid))
```

Comment:
Do we need to consider PlayerType.NOT_BLACKLISTED here? or would g.getPlayerType(uuid) return 'null' for those. Not recollecting atm.